daemon-scheduler/internal/features/steps/step_defns.go

```diff
@@ -655,7 +655,7 @@ func doSomething(ttl time.Duration, tickTime time.Duration, fn func() (bool, err
 }
 
 func filterTasksByStatusRunning(cluster *string, taskARNs []*string, ecsWrapper wrappers.ECSWrapper) []*string {
-	runningTasks := make([]*string, len(taskARNs))
+	runningTasks := make([]*string, 0)
```

Comment:
This was a bug because if you default a slice to say length of 10, the first element that gets appended goes into the 11th spot, not the 1st.